HW3/css/style.css

```
@@ -0,0 +1,400 @@
+  display: block;
+  padding-top: 15px;
+  font: 400 38px/38px 'Conv_9151', sans-serif;
+  letter-spacing: 3px;
+}
+.logo-name {
+  display: block;
+  margin-top: -5px;
+  font: 400 12px/12px 'Roboto', sans-serif;
+}
+.logo-img {
+  display: block;
+  float: left;
+  overflow: hidden;
+}
+.top-nav {
+  height: 100%;
+}
+.top-nav_list {
+  display: flex;
+  margin: 0 auto;
+  width: 622px;
+  height: 100%;
+  justify-content: space-between;
+  align-items: stretch;
+}
+.top-nav_item {
+  color: #fff;
+  line-height: 76px;
+  vertical-align: middle;
+  text-transform: uppercase;
+}
+.top-nav_item:hover {
+  background-color: #fff;
+  color: #2a2a2a;
+  cursor: pointer;
+}
+.switch {
+  color: #fff;
+  background: transparent;
+}
+.switch option{
+  color:#bf1e2e; 
+}
+.switch:after {
+  content: "\25be";
+}
+.slider{
+  position: relative;
+  bottom: -25px;
+  background: url(../img/event/event.png);
+  background-repeat: no-repeat;
+  height: 769px;
+  background-position: center;
+  z-index: 1;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items:flex-end;
+  text-align: center;
+  padding-bottom: 18px;
+  font-family: Roboto;
+}
+.slider_item{
+  height: 93px;
+  width: 92px;
+  background: #4e4e4e;
+  background: url(../img/item_slider.png) center center; 
+  background-repeat: no-repeat;
+  margin:0 0px 18px 0px; 
+}
+.slider_item-active{
+  height: 93px;
+  width: 92px;
+  background: #e14827;
+  margin:0 0px 18px 0px; 
+  background: radial-gradient(circle at top left, transparent 10px, #e14827 0) top left,
+              radial-gradient(circle at top right, transparent 10px, #e14827 0) top right,
+              radial-gradient(circle at bottom right, transparent 10px, #e14827 0) bottom right,
+              radial-gradient(circle at bottom left, transparent 10px, #e14827 0) bottom left;
+  background-size: 50% 50%;
+  background-repeat: no-repeat;
+}
+.slider_item-big{
+  height: 93px;
+  width: 464px;
+  background: radial-gradient(circle at top left, transparent 10px, #e14827 0) top left,
+              radial-gradient(circle at top right, transparent 10px, #e14827 0) top right,
+              radial-gradient(circle at bottom right, transparent 10px, #e14827 0) bottom right,
+              radial-gradient(circle at bottom left, transparent 10px, #e14827 0) bottom left;
+  background-size: 50% 50%;
+  background-repeat: no-repeat;
+  margin:0 0px 18px 1px; 
+  text-align: left;
+}
+.slider_item-left{
+  background: url(../img/left.png) center center;
+  background-color: #4e4e4e;
+  background-repeat: no-repeat;
+  height: 36px;
+  width: 36px;
+  position: relative;
+  top: -46.5px;
+  border-radius:5px;
+  margin-right: 4px;
+}
+.slider_item-right{
+  background: url(../img/right.png) center center;
+  background-color: #4e4e4e;
+  background-repeat: no-repeat;
+  height: 36px;
+  width: 36px;
+  position: relative;
+  top: -46.5px;
+  border-radius:5px;
+  margin-left: 4px;
+}
+.slider_item-left:hover{
```

```suggestion
.slider_item-left:hover,
.slider_item-right:hover{
  background-color: #e14827;
}
```

так простіше, дивись де можна так само згрупувати))